Include/codecs.h

```diff
@@ -53,6 +53,10 @@ PyAPI_FUNC(PyObject *) _PyCodec_Lookup(
 PyAPI_FUNC(int) _PyCodec_Forget(
        const char *encoding
        );
+
+PyAPI_FUNC(int) _PyCodec_Unregister(
```

Comment:
Since PyCodec_Register() is part of the public C API, I suggest to also add unregister to the public C API: name it PyCodec_Unregister() and declare it just after PyCodec_Register().

Without your change, it's excluded from the limited C API. I don't think that we need to exclude it. The API doesn't leak any implementation detail.